File: i2f-jdk/i2f-swl/web/i2f-swl/src/i2f-turbo-web/i2f-swl/core/data/SwlData.js
import SwlContext from "./SwlContext";
import SwlHeader from "./SwlHeader";

/**
 * @return {SwlData}
 */
function SwlData(header = new SwlHeader(), parts = [], attaches = []) {
    /**
     * @type SwlHeader
     */
    this.header = header;
    /**
     * @type String[]
     */
    this.parts = parts
    /**
     * @type {string[]|null}
     */
    this.attaches = attaches
    /**
     * SwlContext
     */
    this.context = new SwlContext()
}


export default SwlData

File: i2f-jdk/i2f-swl/src/main/java/i2f/swl/data/SwlData.java
package i2f.swl.data;

import lombok.Data;

import java.util.List;

/**
 * @author Ice2Faith
 * @date 2024/7/9 20:23
 * @desc
 */
@Data
public class SwlData {
    private SwlHeader header;
    private List<String> parts;
    private List<String> attaches;
    private SwlContext context;
}
